Automattic/jetpack
# Contact form block: Remove unused component

Remove an unused component.

I spotted that an import from `utils/i18n` had slipped through the cracks in https://github.com/Automattic/jetpack/pull/11743. Webpack was used to discover usage and drive the changes in #11743, any imports from missing files should result in errors during build.

This seems to indicate that this component is not a part of the dependency graph and is unused. Simple searching does not reveal any references to the file name.

## Testing
- Confirm the extensions build is successful (yarn build-extensions)
- Confirm that the contact form block includes all the same inner blocks.
